Lib/test/test_signal.py

```diff
@@ -67,6 +67,13 @@ def test_interprocess_signal(self):
         script = os.path.join(dirname, 'signalinterproctester.py')
         assert_python_ok(script)
 
+    def test_valid_signals(self):
+        s = signal.valid_signals()
+        self.assertIsInstance(s, set)
+        self.assertIn(signal.Signals.SIGINT, s)
+        self.assertNotIn(0, s)
+        self.assertNotIn(signal.NSIG, s)
```

Comment:
What do you think of testing len(s) < NSIG? Maybe len(s) <= NSIG?

Reply:
I'm not sure it's very useful.

Reply:
The purpose is to test if NSIG is consistent with sigfillset(), to check if something is really wrong on a platform. I would like to know if range(0, NSIG) miss some signals on some platforms :-)